filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/GcsPathBuilder.scala

```diff
@@ -85,10 +85,22 @@ object GcsPathBuilder {
       if (uri.getScheme == null) PossiblyValidRelativeGcsPath
       else if (uri.getScheme.equalsIgnoreCase(CloudStorageFileSystem.URI_SCHEME)) {
         if (uri.getHost == null) {
-          softBucketParsing(string) map { ValidFullGcsPath(_, uri.getPath) } getOrElse InvalidFullGcsPath(string)
-        } else ValidFullGcsPath(uri.getHost, uri.getPath)
-      } else InvalidScheme(string)
-    } recover { case t => UnparseableGcsPath(string, t) } get
+          softBucketParsing(string) map {
```

Comment:
should probably be called `softObjectParsing` since it actually won't parse buckets any more...

Reply:
Good point. Fixing the names in this 53 version will expand the diff, but will make it easier to read.